### DISCUSSION TITLE
Error in max_results search_all_tweets academic API V2?

### DISCUSSION THREAD

**Discussion by ricotomo** [tweepy/tweepy#1781] [Questions] [answered]
According to the twitter documentation when using the GET /2/tweets/search/all endpoint the max results is 500 "The maximum number of search results to be returned by a request. A number between 10 and the system limit (currently 500). By default, a request response will return 10 results." https://developer.twitter.com/en/docs/twitter-api/tweets/search/api-reference/get-tweets-search-all whereas for the standard API access using the search recent tweets endpoint it is 100. 

However, when I set the max_results parameter to 500 I get the following error: 
tweepy.errors.BadRequest: 400 Bad Request
when requesting `tweet.fields=context_annotations` `max_results` must be less than or equal to `100`

Here is my query: 
tweets = client.search_all_tweets(query=query, tweet_fields=['context_annotations', 'created_at'], start_time="2006-03-27T00:00:00+00:00", end_time=(datetime.datetime.now()-datetime.timedelta(days=1)), max_results=500)

Is this a tweepy bug or an issue with the Twitter API?

**Comment by Harmon758** [comment, marked as answer]
As the error says, `max_results` must be less than or equal to `100` when requesting `tweet.fields=context_annotations`.
This is a [Twitter API limitation](https://twittercommunity.com/t/max-results-and-context-annotations/156427).
